packages/core/src/flex-layout/FlexLayout.css

```diff
@@ -45,17 +49,17 @@
   top: 0;
 }
 .saltFlexLayout-separator-row-start > *:not(:last-child)::after {
-  right: calc(var(--flexLayout-gap) * 1);
+  right: 0;
 }
 .saltFlexLayout-separator-row-center > *:not(:last-child)::after {
-  right: 0;
+  right: calc(var(--flexLayout-gap) * -1);
```

Comment:
The seperator positioning doesn't quite seem to be working as expected in the stroybook. It seems to change it's position based on the combination of `gap` and `seperators="start"/"center"/"end"`